src/main.rs

```diff
@@ -310,6 +317,38 @@ async fn run_server(addr: SocketAddr) -> anyhow::Result<()> {
     Ok(())
 }
 
+/// Spawns a background tokio task which runs all jobs having no schedule
+/// i.e. manually started
+fn spawn_job_oneoffs(ctx: Arc<Context>) {
+    // TODO: Ideally JobSchedule.schedule should become an `Option<Schedule>`
+    // and here we run all those with schedule=None
+    let jobs: Vec<Box<dyn Job + Send + Sync>> = vec![Box::new(PullRequestAssignmentUpdate)];
+
+    for job in jobs {
+        let ctx = ctx.clone();
+        task::spawn(async move {
+            // Allow some spacing between starting jobs
+            let mut interval =
```

Comment:
Shouldn't this be moved outside the loop? So each job is delayed between each other?

Reply:
ok, now the job runner should space a bit each job execution. Added a comment to clarify that there is no guarantee they start in sequence and that the delay is precise. But I think it should do.